test/data/suites/trix/README

```diff
@@ -0,0 +1,41 @@
+This README is taken from the README of the W3C RDF Working Group's
```

Comment:
I think the references in here to w3c should be removed and mentions of TriG should be replaced with TriX

Reply:
README looks good now.